### ISSUE TITLE
Cannot save photo

### ISSUE THREAD

**Issue by dsanchez-at4** [ricardo118/grav-aviary-plugin#10]
When you click the Save button, in order to save the photo, nothing happens but the browser console shows this error:
"There was a problem saving your photo."

In the Network tab on Chrome, I found a request like this:
http://cc-api-aviary-cds.adobe.io/v2/getjobstatus

with the following response:
{"Code":40,"JobStatusCode":"JobFailed","JobStatus":"The job has failed","JobId":"34a6413c-f4a2-4fe4-9533-1b9c49ee731b"}

Does anybody knows what's the problem?

**Comment by ricardo118** [comment]
Yeah unfortunately due to an API change this plugin will no longer work, and adobe have stopped supporting it so unless something changes there's nothing i can do anymore to fix it
